eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -36,28 +66,86 @@ const AttendanceCard: React.FC<Props> = ({
   const endTime = luxonEnd.toISOTime().substring(0, 5);
 
   const timeString = `${startTime} - ${endTime}`;
+
   return (
-    <div>
-      <div data-testid="time-string">{timeString}</div>
-      <div>{categories}</div>
-      {userBookings.map((user) => {
-        const isAbsent = absentees?.includes(user.customer_id) || false;
-        return (
-          <div key={user.customer_id}>
-            <div>{user.name}</div>
-            <button
-              type="button"
-              onClick={() =>
-                dispatch(markAttendance(user.customer_id, isAbsent))
+    <Container maxWidth="sm">
+      <List className={classes.root}>
+        <div key={id + "-" + timeString} className={classes.slotWrapper}>
+          <ListItem className={classes.listHeader}>
+            <ListItemText
+              primary={
+                <span>
+                  {timeString} <b>({userBookings.length})</b>
+                </span>
               }
-            >
-              {isAbsent ? "👎" : "👍"}
-            </button>
-          </div>
-        );
-      })}
-    </div>
+              secondary={translateAndJoinTags(categories, type)}
+            />
+          </ListItem>
+          {userBookings.map((user) => {
+            const isAbsent = absentees?.includes(user.customer_id) || false;
+            const listItemClass = isAbsent ? classes.absent : "";
+
+            const absenteeButtons = (
+              <Button
+                variant="contained"
+                size="small"
+                color={isAbsent ? "primary" : "secondary"}
+                onClick={() =>
+                  dispatch(markAttendance(user.customer_id, isAbsent))
+                }
+                // disabled={hasLocalChange}
+              >
+                {isAbsent ? "👎" : "👍"}
+              </Button>
+            );
+
+            return (
+              <ListItem
+                key={`${id}-${user.customer_id}`}
+                className={listItemClass}
+              >
+                <ListItemAvatar>
+                  <EisbukAvatar {...user} />
+                </ListItemAvatar>
+                <ListItemText primary={user.name} />
+                <ListItemSecondaryAction>
+                  {absenteeButtons}
+                </ListItemSecondaryAction>
+              </ListItem>
+            );
+          })}
+        </div>
+      </List>
+    </Container>
   );
 };
 
+// ***** Start Region Local Utils ***** //
+
+const translateAndJoinTags = (categories: Category[], type: SlotType) => {
+  const translatedCategories = categories.map((category) =>
+    i18n.t(`Categories.${category}`)
+  );
+  const translatedType = i18n.t(`SlotTypes.${type}`);
+
+  return `${[...translatedCategories, translatedType].join(" ")}`;
+};
+// ***** End Region Local Utils ***** //
```

Comment:
Do the regions like this 
```typescript
// #region something

// someCode...

// #endregion something
```
Silvio's shown me this, VSCode lets you contract and expand regions if you do it like this.